pyccel/ast/core.py

```diff
@@ -4941,32 +4941,51 @@ class Slice(Basic):
     end : Symbol or int
         ending index
 
+    step : Symbol or int default None
+
     Examples
     --------
     >>> from sympy import symbols
     >>> from pyccel.ast.core import Slice
-    >>> m, n = symbols('m, n', integer=True)
+    >>> m, n, p = symbols('m, n, p', integer=True)
     >>> Slice(m,n)
     m : n
     >>> Slice(None,n)
      : n
     >>> Slice(m,None)
     m :
+    >>> Slice(m, n, p)
+    m : n : p
```

Comment:
can update the variables names in the doc-string to be more comprehensive.
suggest instead of `m, n, p` use `start, end , step.`